src/app/base/components/record-card/record-card.component.ts

```diff
@@ -0,0 +1,70 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { DataBrowserModals, RecordCommentsModal } from 'src/app/data/components/data-browser/helper-classes/modals-helper';
+
+
+@Component({
+  selector: 'kern-record-card',
+  templateUrl: './record-card.component.html',
+  styleUrls: ['./record-card.component.scss']
+})
+export class RecordCardComponent implements OnInit {
+
+  @Input() record: Record;
+  @Input() attributesSortOrder: AttributeSort;
+  @Input() index: number;
+  @Input() extendedRecords: ExtendedRecord;
+  @Input() similarSearchHelper: any;
+  @Input() recordComments: RecordCommentsModal;
+  @Input() attributes: Map<string, any>;
```

Comment:
if possible let's distance us from Maps a bit (I was the one introducing them however since they use functions (get) in the HTML this might not be optimal). Since we often use strings (uuids/attribute names) as keys they should be interchangeable. Objects keys etc. are more complex but often not necessary.

Also I'm not the biggest fan of the sortOrder/dict combination (Again something I introduced at some point I think) -> I think we should decide on a common approach for attributes throughout the UI.
A: double structure (dict+array+filter of array) -> benefit performance since only a small part is changed + dict access via key is fast
B: only array -> benefit redability -> will be replaced more often (maybe performance hit?) also maybe not since we need to loop anyway?

-> so maybe a global function to prepare an array? (again something we can talk about :) )